addons/account/models/chart_template.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+from __future__ import annotations
```

Comment:
🤔 I don't see any annotations in this file

Reply:
Yes there is, for the `_field_to_sql` function